example/docker/infrasim/start_node.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+CONTAINER=infrasim_1
+INTERFACE=eth1
+BRIDGE=br0
+
+# create the infrasim container
+docker create -it --net=docker_southbound --privileged --name $CONTAINER infrasim/infrasim-compute:3.5.1
+
+# add the admin network to it
+docker network connect docker_admin infrasim_1
+
+# start the container
+docker start $CONTAINER
+
+# copy the vnode config to the container
+docker cp default.yml $CONTAINER:/root/.infrasim/.node_map
+
+# get the IP for the bridge 
+IP=$(docker exec $CONTAINER ifconfig $INTERFACE | awk '/inet addr/{print substr($2,6)}')
```

Comment:
move this line after line24 , together with other network setup in infrasim docker container.